src/modules/components/proposal/proposalActions/proposalActionsTypes/proposalActionUpdateMetadata.ts

```diff
@@ -0,0 +1,51 @@
+import { type IProposalAction } from './proposalAction';
+import { type ProposalActionType } from './proposalActionType';
+
+export interface IMetadataLink {
+    /**
+     * Human readable label for link.
+     */
+    label: string;
+    /**
+     * link href.
+     */
+    href: string;
+    /**
+     * link target.
+     */
+    target: string;
+}
+
+export interface IMetaData {
```

Comment:
I would include the `IProposalActionUpdateMetadata` prefix to the interface to keep the context of proposal-actions, we're doing it for `IProposalActionTokenMintMetadataReceiver` for instance. But `IProposalActionUpdateMetadataMetadata` sounds bad 😅  Maybe `IProposalActionUpdateMetadataDaoMetadata` and `IProposalActionUpdateMetadataDaoMetadataLink`?